city_scrapers/spiders/chi_ssa_60.py

```diff
@@ -0,0 +1,129 @@
+import html
+import json
+from datetime import datetime
+
+from city_scrapers_core.constants import ADVISORY_COMMITTEE, COMMISSION, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.relativedelta import relativedelta
+from scrapy import Request
+
+
+class ChiSsa60Spider(CityScrapersSpider):
+    name = "chi_ssa_60"
+    agency = "Chicago Special Service Area #60 Albany Park"
+    timezone = "America/Chicago"
+    months_backward = 10
```

Comment:
This isn't based on anything in particular, but let's say 6 months backward and 3 months forward?

Reply:
Well, I should've mentioned that looking forward, only October and November are populated on the site currently, so I think we may wind up with some fruitless requests being made by looking 3 months ahead. Probably not a problem, and I'll go ahead and implement it this way, but I figured I should let you know in case you want something different.